### PR TITLE
Mobile Portfolio Form

### PR DESCRIPTION
**Pull request by Thi-Ponce**
### Lesson Objectives:
- Learn how to code and understand the form format;
- Create a responsive form;
- Learn and experience pair programming.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -140,8 +140,27 @@ <h3>LET'S CONNECT</h3>
...
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <ul>
+      <li>
+        <label for="name"></label>
+        <input type="text" name="user_name" id="name" placeholder="Full name" maxlength="30" required>
+      </li>
+      <li>
+        <label for="email"></label>
+        <input type="email" name="user_mail" id="email" placeholder="youremail@email.com" required>
+      </li>
```
Comment (mateo951) [inline index.html:157]: Please try to follow the details on the figma's design 📝 Your inputs for name and email have a red border. What do you think if we change those borders to meet the project requirements 🙌😎

#### Your form 
Captura

#### Figma's 
image

File: index.html [hunk 2]
```diff
@@ -140,8 +140,27 @@ <h3>LET'S CONNECT</h3>
       <li class="language-item"><img src="./Bullets/html.png" alt="html-logo"><p>HTML</p></li>
       <li class="language-item"><img src="./Bullets/css.png" alt="css-logo"><p>CSS</p></li>
       <li>Frameworks<hr></li>
-      <li>Skills</li>
+      <li>Skills<hr></li>
     </ul>
-  </section>
+  </section><br>
+  <form class="form" action="https://formspree.io/f/xdobanld" method="POST">
```
Comment (mateo951) [inline index.html:146]: Nice work applying the formspree functionality 😎 

_The form has been sent successfully_ ✔

image

**Review by mateo951 — CHANGES_REQUESTED** [review]
Hi @Thi-Ponce,

📝👌 **Your project highlights** 🔥😎 
- Linters are on point ✔
- Nice README file 👀
- Good PR 👌
- UI design 🔥

You have done a great job so far 🔥
There is one more issue that you still need to address to go to the next project but you are almost there 😁

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by rotimiazeez — APPROVED** [review]
# Hi Team👋, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🎉🎉
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**